addons/folo/common/src/main/java/org/commonjava/indy/folo/data/DtxTrackingRecord.java

```diff
@@ -0,0 +1,249 @@
+package org.commonjava.indy.folo.data;
+
+
+import com.datastax.driver.mapping.annotations.Column;
+import com.datastax.driver.mapping.annotations.PartitionKey;
+import com.datastax.driver.mapping.annotations.Table;
+import org.commonjava.indy.folo.model.StoreEffect;
+import org.commonjava.indy.folo.model.TrackedContentEntry;
+import org.commonjava.indy.folo.model.TrackingKey;
+import org.commonjava.indy.model.core.AccessChannel;
+import org.commonjava.indy.model.core.StoreKey;
+
+import java.util.Set;
+
+@Table(name = "records" )
```

Comment:
I saw the config for the table name above, do we need to ref that here ?

Reply:
It should be constant. name.
Changed on next commit.